raid/worker.py

```diff
@@ -0,0 +1,45 @@
+import multiprocessing
+import os
+
+
+def _worker_main(queue):
+    """
+
+    :param queue: multiprocessing.Queue
+    :return:
+    """
+    while True:
+        item = queue.get(block=True)
+        func = item['func']
+        callback = item['callback']
+        params = item['params']
+
+        res = func(**params)
+        print(os.getpid(), "got", item)
```

Comment:
shouln't we avoid `print()` in the code, @StasEvseev  ? :-)    use `if _Debug: lg.out(_DebugLevel, ...)` instead?